skimage/util/crop.py

```diff
@@ -0,0 +1,92 @@
+import numpy as np
+
+__all__ = ['crop']
+
+
+def crop(image, bounding_box, axis=None):
```

Comment:
I would also prefer `axes` to `axis` for the variable name since the most common use cases will probably be for 2D and 3D bounding boxes. I know NumPy often uses `axis` for functions that can accept a tuple of axes, but I think that is for historical purposes/backwards compatibility. It seems clearer to just use the plural here

Reply:
changed to axes.